main/test/flix/Test.Exp.Effect.flix

```diff
@@ -100,6 +100,24 @@ namespace Test/Exp/Effect {
         }
     }
 
+    pub def singleEffect(): Bool \ Empty = ???
+
+    pub def complementEffect(): Bool \ ~Empty = ???
+
+    pub def unionEffect(): Bool \ Empty + Fail = ???
+
+    pub def intersectionEffect(): Bool \ Empty & Fail = ???
+
+    pub def differenceEffect(): Bool \ Empty - Fail = ???
```

Comment:
I half buy it. Would it not be better to start simple and then later make it more flexible?

Reply:
I noticed you used the shorthand during the meeting :)

I don't think there's a major difference in complexity.

Reply:
In the paper? 😮

Reply:
I am actually fine with either choice. We can always change it later.

Reply:
> In the paper? open_mouth

At least in something you typed. Not sure if it made it into the paper.

> I am actually fine with either choice. We can always change it later.

Let's keep it like this then, just to greedily avoid work in the short term since we might otherwise change it back.